docs/DeveloperGuide.md

```diff
@@ -121,85 +121,85 @@ testers are expected to do more *exploratory* testing.</div>
 
 ### Adding a recipe
 
-1. **Test case** `add pizza/34/340/eggs/dinner/www.food.com`<br>
-   * **Expected**: Mentioned recipe is added to the list of recipes. <br>
-   Console output providing a brief description of the added recipe <br>
+1. **Test case** `add pizza/34/340/eggs/dinner/www.food.com
+   * **Expected**: Mentioned recipe is added to the list of recipes.
+   Console output providing a brief description of the added recipe 
```

Comment:
Please add a bullet point before console ouput, and use your preview function in your IDE to see the compiled MarkDown. You will see a new bullet point is added for this line, and it looks cleaner.
```suggestion
   * **Console output**: Provides a brief description of the added recipe 
```